src/components/molecules/pager/pager.scss

```diff
@@ -1,10 +1,6 @@
-@import '../../../styles/breakpoints';
-
 .pager__navigation,
 .pager__container {
-	align-items: center;
-	display: flex;
-	justify-content: flex-start;
+	display: block;
```

Comment:
Kate is looking at this on DLP, why are we moving away from flex here?  The _pager is floated and then that float is never cleared.

Reply:
I believe this was totally broken in IE11 when I checked. Once Kate's update is verified this can be reverted or updated.